shell/probe/raindrop.ps1

```diff
@@ -1,46 +1,43 @@
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
 function Run {
     Param([String]$Dat = "")
 
-    $TempFile = [System.IO.Path]::GetTempFileName() | Rename-Item -NewName { [System.IO.Path]::ChangeExtension($_, "exe") } -PassThru
-
-    $Headers = @{
-        'token' = $Token
-        'dos' = $Dos
-        'dat' = $Dat
-    }
     try {
```

Comment:
You can get rid of the try/catch

Reply:
are you confident nothing could throw an unhandled exception in that block?